<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtStart">
       <property name="text">
        <string>Thread 시작</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtStop">
       <property name="text">
        <string>Thread 중지</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtIsRunning">
       <property name="text">
        <string>동작중인지 확인하기</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
